.env.dist

```diff
@@ -8,4 +8,4 @@
 #
 # To enable them in order to provide a fix, set to "on".
 #
-APC_ENABLE_CLI=off
+APC_ENABLE_CLI=on
```

Comment:
Still exists a reason to disable APC test as this PR provide the fix.

So I suggest removing this flag, and always enable it.

Reply:
No sure I follow. The point of this PR is precisely to fix the issues that have been preventing the APC tests from running as they should. What else is missing?

I would actually suggest removing that setting altogether and resume always running the APC tests as I don't see the need for the setting once the APC tests are fixed (by this PR). If APC/APCu is not installed, the tests just skip.

Reply:
> I would actually suggest removing that setting altogether and resume always running the APC tests

Yes, you can do it.

Reply:
I've removed `APC_ENABLE_CLI` and have set `apc.enable_cli=1` so that the APC tests always run.